addons/payment_mollie/models/payment_transaction.py

```diff
@@ -120,6 +120,8 @@ def _process_notification_data(self, notification_data):
             self._set_authorized()
         elif payment_status == 'paid':
             self._set_done()
+        elif payment_status == 'open':
```

Comment:
IMO, 'open' corresponds to 'pending' in Odoo, as we use this state to represent that the provider has received (but not completed) the payment.